examples/server/server.cpp

```diff
@@ -3602,15 +3625,31 @@ int main(int argc, char ** argv) {
         }
     };
 
-    const auto handle_props = [&ctx_server, &res_ok](const httplib::Request &, httplib::Response & res) {
+    std::mutex chat_templates_mutex;
```

Comment:
Can we avoid this `mutex`? Maybe get the chat templates once at the start and store them in the server context.

Reply:
Done, thanks!